config/routes.rb

```diff
@@ -24,4 +24,5 @@
   resources :contact_details, only: [:update]
   resources :abouts, only: [:index, :update]
   resources :services
+  resources :contact_messages
```

Comment:
@arps1989dev we have only create API for `contact message`, why you take resource
Change this to `resources :contact_messages, only: [:create]`